UpdateMunicipalities/handler.ts

```diff
@@ -17,14 +21,45 @@ const logPrefix = "getUpdateMunicipalitiesHandler";
 export const getUpdateMunicipalitiesHandler = (
   blobService: BlobService
 ) => async (context: Context): Promise<unknown> => {
-  return sequenceT(taskEither)(
+  const partialResults = sequenceT(taskEitherSeq)(
```

Comment:
I've to refactor this way (but it's not the only way I think), in order to avoid Task maximum number exceeded. All feedbacks are welcome!